.gitmodules

```diff
@@ -1,12 +1,3 @@
-[submodule "edk2_tici"]
```

Comment:
FYI @robbederks

We can setup a private builder repo with these, but I don't think it'll be too bad like this.